package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "build": "webpack -d --watch",
-    "seed-db": "mongoimport --db restaurants --collection Nearby --file nearby-data.json --jsonArray",
+    "seed-db": "mongoimport --db carousels --collection Carousel --file carousel-data.json --jsonArray",
     "start": "nodemon server/index.js",
```

Comment:
npm start should run just "node server/index.js".
you can create start:dev for nodemon.